apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -1,13 +1,14 @@
 import React, {PropTypes} from 'react';
-import {personalProjectDataPropType} from './projectConstants';
 import PersonalRecentProjects from './PersonalRecentProjects.jsx';
 import NewProjectButtons from './NewProjectButtons.jsx';
 import ContentContainer from '../ContentContainer.jsx';
 import i18n from "@cdo/locale";
+import _ from 'lodash';
 
 const ProjectWidget = React.createClass({
   propTypes: {
-    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
+    projectList: PropTypes.array,
```

Comment:
hi @markabarrett , yes, it would be great if we could show you a few places where the display stays partially rendered while data is loading from the network, and talk about what a good UX would be for those situations.